### ISSUE TITLE
Firefox: Details/summary expanding and collapsing actions not voiced

### ISSUE THREAD

**Issue by juleskuehn** [nvaccess/nvda#8631]
#### Have you tried any other versions of NVDA? No

[Other test cases and results related to detail/summary support with screen readers and browsers](http://wet-boew.github.io/wet-boew-documentation/assessment/2018-1.html)

**Comment by DrSooom** [comment]
Same result with NVDA 2018.1 and Firefox 56.0.2. Changing this value is correctly updated in braille, but not spoken.

**Comment by juleskuehn** [comment]
This also happens with Chrome 68 on [a different test case](http://wet-boew.github.io/wet-boew-documentation/testcase/2018-7.html).

**Comment by grahamarmfield** [comment]
This is still occurring in NVDA 2018.4 with Firefox 64 on Windows 10.

Compare how NVDA behaves when aria-expanded value on a button toggles from 'false' to 'true' and back again. 

This would be a useful enhancement in how NVDA functions - facilitating the use of a native way of implementing accordions without relying on scripting and aria.

**Comment by grahamarmfield** [comment]
I have just retested this with NVDA 2019.2 and the issue is still occurring - ie that the 'expanded' and 'collapsed' state changes are not announced when users interact with the `<summary>` and `<details>` elements.

Are there any plans to update NVDA functionality to dynamically announce the state changes on the `<summary>` and `<details>` elements?

**Comment by amandarush** [comment]
I've tested this using Firefox 68.0.2 and am getting the same reported result, namely that expand/colapse of <summary is not spoken. I agree with Graham that having this spoken would make accordians much easier to create, with less JS and Aria, which would make thhem a lot easier to make accessible.

**Comment by amandarush** [comment]
I am using: NVDA 2019.2. Forgot to add that to the previous comment.

**Comment by Adriani90** [comment]
cc: @jcsteh, @MarcoZehe

**Comment by jcsteh** [comment]
This is a bug in Firefox which is currently being worked on. https://bugzilla.mozilla.org/show_bug.cgi?id=1277201

**Comment by mfranzke** [comment]
The latest commit related to that mozilla bugzilla ticket sounds promising !

**Comment by jcsteh** [comment]
Yes, this is fixed for Firefox 71, available now to test in Firefox Nightly.
